### PR TITLE
WebRTC Protobuf & Shared Code Update 

### PR DESCRIPTION
**Pull request by Gaukas**
Adding WebRTC-related fields to protobuf. Changes are not final.

### CODE REVIEW

**Review by Gaukas — COMMENTED** [review]
- Added webrtc capability to shared code.
- Modified Protobuf to support webrtc.

File: protobuf/signalling.proto [hunk 1]
```diff
@@ -141,6 +141,25 @@ enum TransportType {
...
+    optional uint32 sdp_type = 1; // TODO: Single byte (uint8)
+    // the IP is represented as a 16-byte datatype.
+    optional uint64 ip_upper = 2; // ip := upper << 64 | lower
+    optional uint64 ip_lower = 3;
+    optional uint32 composed_info = 4;
+}
+
+message WebrtcRandseed {
+    optional string seed = 1;
+    optional string shared_secret = 2;
```
Comment (ewust) [inline protobuf/signalling.proto:157]: We should probably not send the shared secret in the clear like that, no? Is this derived from the seed? If so, then no need to send it in this protobuf

File: tapdance/conjure.go [hunk 2]
```diff
@@ -292,6 +292,8 @@ func DialConjure(ctx context.Context, cjSession *ConjureSession, registrationMet
 
 	cjSession.setV6Support(both)
 
+	// TODO: WebRTC Init Connection and get LocalSDP, send with Register()
```
Comment (ewust) [inline tapdance/conjure.go:295]: Probably should not merge this whole PR until we have the rest completed; we can merge the protobuf things separately for now

**Review by ewust — CHANGES_REQUESTED** [review]
Let's trim this down to just the protobuf changes for now, and leave the TODOs/active dev stuff on a branch.

**Pull request closed without merging**